### ISSUE TITLE
Rendering issue because of wrong built path from resource links

### ISSUE THREAD

**Issue by cderv** [quarto-dev/quarto-cli#4593]
I have a similar problem (to #4510 maybe)

I am updating my syllabus which I prepared with Quarto using Rstudio. There was a hyperlink that wasn't working.

I changed it. But somehow, when I render the document, the following happens:

``` 
The filename, directory name, or volume label syntax is incorrect. (os error 123), stat 
'D:\Onedrive\R\projects\kjayhan.github.io\syllabi\(https:\www.nyu.edu\about\policies-guidelines-compliance\policies-and-guidelines\academic-integrity-for-students-at-nyu.html)'
``` 
It seems that somehow the link got mixed up into a syntax, but how? And how to solve this? Any ideas?

_Originally posted by @kjayhan in https://github.com/quarto-dev/quarto-cli/issues/4510#issuecomment-1451258285_

**Comment by cderv** [comment]
>  am updating my syllabus which I prepared with Quarto using Rstudio. There was a hyperlink that wasn't working.

@kjayhan can you share more about the change you've made ?  I understand this was working and changing to a url makes the error ? 

> (https:\www.nyu.edu\about\policies-guidelines-compliance\policies-and-guidelines\academic-integrity-for-students-at-nyu.html)

Where do you set this hyperlink in the document or project ? Can you provide anything that would help us reproduce on our end ? 

At least can you share a traceback following this steps: https://quarto.org/bug-reports.html#dont-hold-back-tell-us-anything-you-think-might-make-a-difference

thank you

**Comment by kjayhan** [comment]
It was written like this:


```
["Plagiarism: presenting others' work without adequate acknowledgement of its source, as though it were one's own. Plagiarism is a form of fraud. We all stand on the shoulders of others, and we must give credit to the creators of the works that we incorporate into products that we call our own. Some examples of plagiarism:]((https://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/academic-integrity-for-students-at-nyu.html))

-   [a sequence of words incorporated without quotation marks;]((https://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/academic-integrity-for-students-at-nyu.html))

-   [an unacknowledged passage paraphrased from another's word;]((https://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/academic-integrity-for-students-at-nyu.html))

-   [the use of ideas, sound recordings, computer data or images created by others as though it were one's own."]((https://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/academic-integrity-for-students-at-nyu.html))
```

I first deleted the additional ( in ((.

It created the problem. Then, I changed it to

```

"Plagiarism: presenting others' work without adequate acknowledgement of its source, as though it were one's own. Plagiarism is a form of fraud. We all stand on the shoulders of others, and we must give credit to the creators of the works that we incorporate into products that we call our own. Some examples of plagiarism:]

-   a sequence of words incorporated without quotation marks;

-   an unacknowledged passage paraphrased from another's word;

-   the use of ideas, sound recordings, computer data or images created by others as though it were one's own." [Source](https://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/academic-integrity-for-students-at-nyu.html).
```

This also did not help.

I went back to the original version. But the problem persisted.

But I think this is a Windows problem. I am now on my Mac, and everything works now.

**Comment by cscheid** [comment]
I see a couple of markdown syntax issues, and I wonder if this is confusing our code.

>     ["Plagiarism: presenting others' work without adequate acknowledgement of its source, as though it were one's own. Plagiarism is a form of fraud. We all stand on the shoulders of others, and we must give credit to the creators of the works that we incorporate into products that we call our own. Some examples of plagiarism:]((https://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/academic-integrity-for-students-at-nyu.html))

There are two things going on here at least. First, there is a stray " in the text content. Second, as you noticed, there's two parentheses `((` `))` instead of one. Can you try removing that starting quote to see if the problem goes away?

To be clear, our code shouldn't crash here! But if this solves the problem, then it's a clue for us.

**Comment by kjayhan** [comment]
> I see a couple of markdown syntax issues, and I wonder if this is confusing our code.
> 
> > ```
> > ["Plagiarism: presenting others' work without adequate acknowledgement of its source, as though it were one's own. Plagiarism is a form of fraud. We all stand on the shoulders of others, and we must give credit to the creators of the works that we incorporate into products that we call our own. Some examples of plagiarism:]((https://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/academic-integrity-for-students-at-nyu.html))
> > ```
> 
> There are two things going on here at least. First, there is a stray " in the text content. Second, as you noticed, there's two parentheses `((` `))` instead of one. Can you try removing that starting quote to see if the problem goes away?
> 
> To be clear, our code shouldn't crash here! But if this solves the problem, then it's a clue for us.

I figured it may be because of the quote, and got rid of it as in the following:

> "Plagiarism: presenting others' work without adequate acknowledgement of its source, as though it were one's own. Plagiarism is a form of fraud. We all stand on the shoulders of others, and we must give credit to the creators of the works that we incorporate into products that we call our own. Some examples of plagiarism:]
> 
> -   a sequence of words incorporated without quotation marks;
> 
> -   an unacknowledged passage paraphrased from another's word;
> 
> -   the use of ideas, sound recordings, computer data or images created by others as though it were one's own." [Source](https://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/academic-integrity-for-students-at-nyu.html).

I put the link after the quote, by adding the word [Source].

It didn't solve the problem.

But as I said previously, it worked just fine (as it used to) on my Mac. It was/ is a Windows issue I think.

**Comment by cscheid** [comment]
That doesn't solve a potential bug source for us, because the second quotation mark is in a different Pandoc element, and so they still don't nest properly (and still potentially confuse our codebase). Can you try removing those two quote marks? Thanks.

**Comment by cderv** [comment]
> But as I said previously, it worked just fine (as it used to) on my Mac. It was/ is a Windows issue I think.

For context @kjayhan, yes this is a windows issue and flagged as such. We just need if possible more of your help so that we can find what goes wrong here. You definitely found a discrepancy that should be fixed on our hand! Thank you very much

**Comment by kjayhan** [comment]
I tried to reproduce the error on my Windows computer. But before doing it, I tried rendering the corrected version that already worked on my Mac. It still throws the same error. Some problem got stuck, and cannot move forward:

> 
> Output created: ..\docs\syllabi\ERROR REPRODUCE.html
> 
> Terminating existing preview server....DONE
> 
> Preparing to preview
> ERROR: The filename, directory name, or volume label syntax is incorrect. (os error 123), stat 'D:\Onedrive\R\projects\kjayhan.github.io\(https:\www.nyu.edu\about\policies-guidelines-compliance\policies-and-guidelines\academic-integrity-for-students-at-nyu.html)'

Now, adding or removing two quote marks would not seem to change anything.

FYI, the current version that works on Mac, but not Windows is as follows:


> "Plagiarism: presenting others' work without adequate acknowledgement of its source, as though it were one's own. Plagiarism is a form of fraud. We all stand on the shoulders of others, and we must give credit to the creators of the works that we incorporate into products that we call our own. Some examples of plagiarism:]
> 
> -   a sequence of words incorporated without quotation marks;
> 
> -   an unacknowledged passage paraphrased from another's word;
> 
> -   the use of ideas, sound recordings, computer data or images created by others as though it were one's own." [Source](https://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/academic-integrity-for-students-at-nyu.html).

**Comment by cderv** [comment]
@kjayhan can you help me understand your file organization and quarto location ? 

> 'D:\Onedrive\R\projects\

It seems you have your project file on D: - can you confirm me where is quarto install ? 

* Open powershell and type `gcm quarto`
* Also type `quarto check` and `quarto --paths` 

Thank you !

**Comment by cderv** [comment]
Currently I did try your example 

````markdown
---
title: "test"
---

"Plagiarism: presenting others' work without adequate acknowledgement of its source, as though it were one's own. Plagiarism is a form of fraud. We all stand on the shoulders of others, and we must give credit to the creators of the works that we incorporate into products that we call our own. Some examples of plagiarism:]

-   a sequence of words incorporated without quotation marks;

-   an unacknowledged passage paraphrased from another's word;

-   the use of ideas, sound recordings, computer data or images created by others as though it were one's own." [Source](https://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/academic-integrity-for-students-at-nyu.html).
````

also on ONEDRIVE folder, and it does work for me without issue. 

You could try also move your file out of onedrive to see if this could be the issue, and provide us with reproducible steps if possible.

Anyhow, the information asked before will help (my OneDrive folder is on C:, like other files on my system). Yours may not be the same

**Comment by kjayhan** [comment]
I could not recreate the problem, sorry. After I corrected the hyperlink, the problem persisted. Rename the qmd file solved the problem (I think because of the mistake in the earlier hyperlink, it created a file/ folder name with D:\Onedrive\R\projects\kjayhan.github.io(https:\www.nyu.edu\about\policies-guidelines-compliance\policies-and-guidelines\academic-integrity-for-students-at-nyu.html)'. After renaming the qmd file, the problem was solved.
